pkg/querier/block.go

```diff
@@ -55,7 +55,7 @@ func NewBlockQuerier(cfg tsdb.Config, logLevel logging.Level, r prometheus.Regis
 	level.Info(util.Logger).Log("msg", "successfully synchronized TSDB blocks for all users")
 
 	stopc := make(chan struct{})
-	go runutil.Repeat(30*time.Second, stopc, func() error {
+	go runutil.Repeat(cfg.BucketStore.SyncInterval, stopc, func() error {
```

Comment:
There should probably be some validation of `SyncInterval` being >= 0.

Reply:
Right, done.